aten/src/ATen/function_wrapper.py

```diff
@@ -1607,6 +1611,44 @@ def process_native(option):
     return type_object_declarations, type_object_definitions
 
 
+def process_extension_backend_option(option):
+    # type: (FunctionOption) -> FunctionOption
+    option['formals_types'] = [f['type'] for f in option['formals_list']]
+    option['native_actuals'] = [f['name'] for f in option['formals_list']]
+    schema_args = ", ".join(
+        ["{} {}".format(f['dynamic_type'], f['name']) for f in option['formals_list']])
+    return_type = NATIVE_DYNAMIC_TYPE.get(option['return_type'], option['return_type'])
+    option['schema'] = "{}({}) -> {}".format(option['api_name'], schema_args, return_type)
```

Comment:
is this the only place we'll need to replace when we get the real schema?  Can you mark them with a TODO / NOTE so I can easily find it/them in the future?